test/integration/tcpingress_test.go

```diff
@@ -36,7 +36,6 @@ var (
 )
 
 func TestTCPIngressEssentials(t *testing.T) {
```

Comment:
```suggestion
func TestTCPIngressEssentials(t *testing.T) {
    RunWhenKongExpressionRouterWithVersion(t, ">=3.4.0")
```
Need a further version check here because only Kong 3.4 and newer can support it.